news/views.py

```diff
@@ -0,0 +1,164 @@
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.shortcuts import render
+from django.urls import reverse_lazy
+from django.views import generic
+
+from news.forms import (
+    NewspaperCreateForm,
+    NewspaperSearchForm,
+    RedactorCreationForm,
+    RedactorSearchForm,
+    RedactorYearsUpdateForm,
+    TopicSearchForm,
+)
+from news.models import Newspaper, Redactor, Topic
+
+
+def index(request):
+    num_redactors = Redactor.objects.count()
+    num_newspapers = Newspaper. objects.count()
+    num_topics = Topic.objects.count()
+
+    context = {
+        "num_redactors": num_redactors,
+        "num_newspapers": num_newspapers,
+        "num_topic": num_topics
+    }
+
+    return render(request, "news/index.html", context=context)
+
+
+class TopicListView(generic.ListView):
+    model = Topic
+    paginate_by = 10
+
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(TopicListView, self).get_context_data(**kwargs)

```

Comment:
```suggestion
        context = super(TopicListView, self).get_context_data(**kwargs)
```suggestion
        context = super().get_context_data(**kwargs)

```
```